Latest commit broke `--pull-params`

[The latest commit for main.js](https://github.com/mosteo-actions/docker-run/commit/fc9393a233449b9eef824f55effa687d038d8efb) has dropped the `--pull-params` argument from #5, which causes multipath-tools CI runs to fail.

Sample excerpt from [CI run](https://github.com/openSUSE/multipath-tools/actions/runs/7913147635/job/22019820547#step:8:68):

```
2024-02-27T08:20:59.8484457Z ##[debug]Starting: run tests
2024-02-27T08:20:59.8510266Z ##[debug]Loading inputs
2024-02-27T08:20:59.8511865Z ##[debug]Evaluating: format('ghcr.io/mwilck/multipath-run-debian-{0}', matrix.os)
2024-02-27T08:20:59.8512501Z ##[debug]Evaluating format:
2024-02-27T08:20:59.8512858Z ##[debug]..Evaluating String:
2024-02-27T08:20:59.8513441Z ##[debug]..=> 'ghcr.io/mwilck/multipath-run-debian-{0}'
2024-02-27T08:20:59.8514044Z ##[debug]..Evaluating Index:
2024-02-27T08:20:59.8514531Z ##[debug]....Evaluating matrix:
2024-02-27T08:20:59.8514924Z ##[debug]....=> Object
2024-02-27T08:20:59.8515297Z ##[debug]....Evaluating String:
2024-02-27T08:20:59.8515776Z ##[debug]....=> 'os'
2024-02-27T08:20:59.8516243Z ##[debug]..=> 'bookworm'
2024-02-27T08:20:59.8516728Z ##[debug]=> 'ghcr.io/mwilck/multipath-run-debian-bookworm'
2024-02-27T08:20:59.8517431Z ##[debug]Result: 'ghcr.io/mwilck/multipath-run-debian-bookworm'
2024-02-27T08:20:59.8518290Z ##[debug]Evaluating: github.workspace
2024-02-27T08:20:59.8518834Z ##[debug]Evaluating Index:
2024-02-27T08:20:59.8519226Z ##[debug]..Evaluating github:
2024-02-27T08:20:59.8519579Z ##[debug]..=> Object
2024-02-27T08:20:59.8520042Z ##[debug]..Evaluating String:
2024-02-27T08:20:59.8520426Z ##[debug]..=> 'workspace'
2024-02-27T08:20:59.8520985Z ##[debug]=> '/home/runner/work/multipath-tools/multipath-tools'
2024-02-27T08:20:59.8521708Z ##[debug]Result: '/home/runner/work/multipath-tools/multipath-tools'
2024-02-27T08:20:59.8522768Z ##[debug]Evaluating: format('--platform linux/{0}', env.CONTAINER_ARCH)
2024-02-27T08:20:59.8523358Z ##[debug]Evaluating format:
2024-02-27T08:20:59.8523828Z ##[debug]..Evaluating String:
2024-02-27T08:20:59.8524251Z ##[debug]..=> '--platform linux/{0}'
2024-02-27T08:20:59.8524666Z ##[debug]..Evaluating Index:
2024-02-27T08:20:59.8525127Z ##[debug]....Evaluating env:
2024-02-27T08:20:59.8525521Z ##[debug]....=> Object
2024-02-27T08:20:59.8525848Z ##[debug]....Evaluating String:
2024-02-27T08:20:59.8526313Z ##[debug]....=> 'CONTAINER_ARCH'
2024-02-27T08:20:59.8528484Z ##[debug]..=> 'ppc64le'
2024-02-27T08:20:59.8529075Z ##[debug]=> '--platform linux/ppc64le'
2024-02-27T08:20:59.8529598Z ##[debug]Result: '--platform linux/ppc64le'
2024-02-27T08:20:59.8530622Z ##[debug]Evaluating: format('--platform linux/{0}', env.CONTAINER_ARCH)
2024-02-27T08:20:59.8531288Z ##[debug]Evaluating format:
2024-02-27T08:20:59.8531693Z ##[debug]..Evaluating String:
2024-02-27T08:20:59.8532065Z ##[debug]..=> '--platform linux/{0}'
2024-02-27T08:20:59.8532608Z ##[debug]..Evaluating Index:
2024-02-27T08:20:59.8533018Z ##[debug]....Evaluating env:
2024-02-27T08:20:59.8533396Z ##[debug]....=> Object
2024-02-27T08:20:59.8533801Z ##[debug]....Evaluating String:
2024-02-27T08:20:59.8534219Z ##[debug]....=> 'CONTAINER_ARCH'
2024-02-27T08:20:59.8534684Z ##[debug]..=> 'ppc64le'
2024-02-27T08:20:59.8535218Z ##[debug]=> '--platform linux/ppc64le'
2024-02-27T08:20:59.8535643Z ##[debug]Result: '--platform linux/ppc64le'
2024-02-27T08:20:59.8536181Z ##[debug]Loading env
2024-02-27T08:20:59.8545598Z ##[group]Run mosteo-actions/docker-run@v1
2024-02-27T08:20:59.8546008Z with:
2024-02-27T08:20:59.8546805Z   image: ghcr.io/mwilck/multipath-run-debian-bookworm
2024-02-27T08:20:59.8547453Z   guest-dir: /build
2024-02-27T08:20:59.8547925Z   host-dir: /home/runner/work/multipath-tools/multipath-tools
2024-02-27T08:20:59.8548601Z   command: -C tests
2024-02-27T08:20:59.8548964Z   params: --platform linux/ppc64le
2024-02-27T08:20:59.8549407Z   pull-params: --platform linux/ppc64le
2024-02-27T08:20:59.8550067Z env:
2024-02-27T08:20:59.8550379Z   CONTAINER_ARCH: ppc64le
2024-02-27T08:20:59.8550732Z ##[endgroup]
```

Above you can see that the workflow is asking for a `linux/ppc64le` container, but docker tries to download `linux/amd64` and fails:

```
2024-02-27T08:21:00.1493673Z Host PATH: /snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
2024-02-27T08:21:00.1667046Z Using default tag: latest
2024-02-27T08:21:00.7149001Z latest: Pulling from mwilck/multipath-run-debian-bookworm
2024-02-27T08:21:00.7152842Z no matching manifest for linux/amd64 in the manifest list entries
2024-02-27T08:21:00.7211414Z ##[error]Error: stdout Using default tag: latest
latest: Pulling from mwilck/multipath-run-debian-bookworm
, stderr no matching manifest for linux/amd64 in the manifest list entries
```

--- a/main.js
+++ b/main.js
@@ -19,9 +19,10 @@ function run() {
             const guestDir = core.getInput('guest-dir');
             const command = core.getInput('command');
             const params = core.getInput('params');
+            const pull_params = core.getInput('pull-params');
             core.info(`Host PATH: ${process.env.PATH}`);
             // pull the required machine
-            yield dockerCommand(`pull ${image}`);
+            yield dockerCommand(`pull ${pull_params} ${image}`);
             core.info(`Pulled OK: ${image}`);
             // run it
             yield dockerCommand(`run ${params} -w ${guestDir} -v${hostDir}:${guestDir} ${image} ${command}`);

--- a/src/main.ts
+++ b/src/main.ts
@@ -9,11 +9,12 @@ async function run() {
       const guestDir : string = core.getInput('guest-dir');
       const command  : string = core.getInput('command');
       const params   : string = core.getInput('params');
+      const pull_params   : string = core.getInput('pull-params');
 
       core.info(`Host PATH: ${process.env.PATH}`);
 
       // pull the required machine
-      await dockerCommand(`pull ${image}`);
+      await dockerCommand(`pull ${pull_params} ${image}`);
       core.info(`Pulled OK: ${image}`);
 
       // run it

